### DISCUSSION TITLE
When will 2.0 go beta?

### DISCUSSION THREAD

**Discussion by gollenia** [timber/timber#2474] [General]
Hi,

I use Timber a lot for themes and plugins. It seems that development of 2.0 has stagnated in the last months. Is it still in active development? When will 2.0 hit the masses? I installed it for some of my projects, and it works like a charm, but running on production with an alpha makes me feel a bit uncomfortable...

c ya, thomas

**Comment by gchtr** [comment]
Hey @gollenia 

Yes, 2.0 is still in active development. You’re right that it seems like work has stagnated. This is mainly because we haven’t found as much time to work on it as we wanted to. We’re still motivated, but struggle to find slots in our free time to add the final touches for the 2.0 release. We’re very sorry to have to keep you waiting, but personal life and health still have a higher priority for us :). I’m sure you can understand that, too.

So, even though it might work like a charm, I still wouldn’t run it in production just yet. Unfortunately, we can’t set a release date yet, either. But we will get there!

**Reply by gollenia** [reply]
Thanks for your reply! Glad to see that things are going on. I absolutely underrstand that these kind of projects take a lot of time. I love twig, and am thankful for your work. I tried to code my own twig-bridge many years ago and was so happy when i found Timber. I've been using it for a long time now. I would offer to help you, but I guess it's not easy to step into an already well working team with it's own work flows and coding guidelines. But I will pray for your personal health and life and still alittle energy for Timber.

**Reply by gchtr** [reply]
@gollenia Thanks for your kind words and for understanding! 

> I would offer to help you, but I guess it's not easy to step into an already well working team with it's own work flows and coding guidelines.

Well, I wouldn’t worry too much about our workflows and coding guidelines if you want to help out. It definitely shouldn’t hold you back from making a pull request. You can always start small and then slowly grow into it. And in case you need help, we can always guide you through it.

**Comment by xavivars** [comment]
Hey @gchtr, 

I'm building a new website using latest 1.x, and I would hate to have to redo it soon because 2.0 is not there yet.

I have some free time over the next few weeks, I've contributed (just a bit!) to Timber in the past, and I would love to help if there's anything I could work on.

Is there any area where you'd need help to make 2.0 (or at least, 2.0 beta) a reality? Maybe @jarednova has ideas too...

**Reply by jarednova** [reply]
Hey man, great to hear from you again! The best help is testing testing testing. Yes, we have PHPUnit but that doesn't tell us what it's like or some of the ways "real" setups interact with Timber & Twig. Beyond that, feel free to browse the `Help Wanted` tags in Issues.

**Comment by gchtr** [comment]
The first beta for Timber version 2 is here is now: https://github.com/timber/timber/releases/tag/2.0.0-beta.1

Thank you for being so patient and understanding.

We’re happy if you can test it out and report any bugs. Or if you feel like tackling known bugs for 2.0, [here’s a list of issues](https://github.com/timber/timber/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A2.0) tagged with 2.0.

**Comment by Levdbas** [comment]
Hi everyone, the team is hard at work to make Timber 2.0 happen. We've just created an updated roadmap with open tickets, pull requests, etc. If you would like to help out on Timber, now is the perfect timing. See https://github.com/timber/timber/issues/2741

Closing this ticket in favor of the mentioned ticket above.
